### PR TITLE
added task 5

### PR DESCRIPTION
**Pull request by Darina007**

### CODE REVIEW

File: task5/servlets/CheckServlet.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/check")
+public class CheckServlet extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
```
Comment (smiakchilo) [inline task5/servlets/CheckServlet.java]: Always use `@Override`  annotation with the methods that - well - override methods from a superclass)

File: task5/servlets/CheckServlet.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            throws IOException {
+        PrintWriter pw = response.getWriter();
+        response.setContentType("application/json");
+        pw.println("{\"success\" : true}");
+        pw.flush();
+    }
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```
Comment (smiakchilo) [inline task5/servlets/CheckServlet.java]: Empty method not needed here

File: task5/servlets/NameServlet.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+@WebServlet("/get")
+public class NameServlet extends HttpServlet {
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+        String name = request.getParameter("name");
+        PrintWriter pw = response.getWriter();
+
+        pw.println("<html>");
+        pw.println("<h1 '>Name is { " + name + " } </h1>");
```
Comment (smiakchilo) [inline task5/servlets/NameServlet.java]: Here's an error in the tag

File: task5/servlets/StatusServlet.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/status")
+public class StatusServlet extends HttpServlet {
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
```
Comment (smiakchilo) [inline task5/servlets/StatusServlet.java]: `@Override` missing

File: task5/servlets/StatusServlet.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        PrintWriter writer = response.getWriter();
+        String text = "Application Is Running";
+        try {
+            writer.println("<h2><font color='red' face='Arial'>" + text + "</font></h2>");
+        } finally {
+            writer.close();
+        }
+    }
+
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
```
Comment (smiakchilo) [inline task5/servlets/StatusServlet.java]: Redundant method

File: task5/servlets/Test1ForwardServlet.java [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+
+@WebServlet("/test1")
+public class Test1ForwardServlet extends HttpServlet {
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
```
Comment (smiakchilo) [inline task5/servlets/Test1ForwardServlet.java]: Same here

File: task5/servlets/Test1ForwardServlet.java [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+            throws IOException, ServletException {
+        String path = "/status";
```
Comment (smiakchilo) [inline task5/servlets/Test1ForwardServlet.java]: Please take on the notion of constant. Constants are invariable strings, or numbers, etc. that are just "they are as they are". Constants are declared at the beginning of the file like `private static final String PATH = "/status";`

File: task5/servlets/NameServlet.java (hunk 3 above)
Comment (smiakchilo) [inline task5/servlets/NameServlet.java]: Name length condition check is not there

**Review by smiakchilo — CHANGES_REQUESTED** [review]
Please remove files not related to Task 5 from the pull request. Then feel free to merge

**Comment by Darina007** [comment]
I apologize. It seems I did something wrong and accidentally closed the pull request. I will create a new one and make changes there, and upon your approval, I will delete this one.

**Comment by smiakchilo** [comment]
Ok)

**Pull request closed without merging**